### DISCUSSION TITLE
Using the Chrome AI Summarizer (Early Look)

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#36] [Comments]
# Using the Chrome AI Summarizer (Early Look)

A look at Chrome's continued development of on-device Gen AI

https://www.raymondcamden.com/2024/09/10/using-the-chrome-ai-summarizer-early-look

**Comment by paulirish** [comment]
Heya Ray. This is awesome stuff. I'll happily be the one to ask to open up the rsstojson feed service.

Mostly, I'm very familiar with my own blog posts so seeing a summary of them would be easier for me to evaluate than a summary of someone else's. :)

Nice work on this!

**Reply by cfjedimaster** [reply]
Done, let me know. As an FYI, the usual rssparser npm module I use does *not* work on Cloudflare, so I'm doing it by hand and it's not been heavily tested. You can see the source here: https://github.com/cfjedimaster/cloudflareworkers-demos/tree/main/rsstojson

Glad you like it. :)
